File: Logic-1/sortaSum/SortaSum.java
package sortaSum;

public class SortaSum {
    //Retorne a soma entre "a" e "b", porém se a soma estiver no intervalo de 10 a 19, inclusive, então retorne 20.

    public int sortaSum(int a, int b){
        if (a+b >= 10 && a+b <= 19) return 20;
        else return a+b;
    }

}

File: Python/Logic-1/sorta_sum.py
"""
Dado 2 números inteiros ("a" e "b"), retorne a sua soma, exceto quando ela estiver
no intervalo entre 10..19, inclusive, nesse caso retorne 20.
"""


def sorta_sum(a, b):
    return 20 if 10 <= a+b <= 19 else a+b


print(sorta_sum(11, 4))
